build/Common.prod.props

```diff
@@ -3,7 +3,12 @@
 
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <DefaultTargetFrameworks>net6.0;netstandard2.0;net462</DefaultTargetFrameworks>
+    <TargetFrameworksForLibraries>net6.0;netstandard2.0;net462</TargetFrameworksForLibraries>
+    <TargetFrameworksForLibrariesExtended>net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworksForLibrariesExtended>
```

Comment:
curious - what's the rationale behind this naming? (I saw it's used for OTLP exporter and OpenTelemetry project.)

Reply:
Just a name for libs that need to target extra netstandard TFM.
My hope is that after this PR:
1. All the TFMs are centrally managed in this single file. It'll help us to manage / review TFMs in the future (e.g. when we add net8 or retire net7), and stop bleeding as new libs being added.
2. We will do the same for tests/examples.

Reply:
Possible to do this here...

`<TargetFrameworksForLibrariesExtended>$(TargetFrameworksForLibraries);netstandard2.1</TargetFrameworksForLibrariesExtended>`

...or order needs to be preserved?

Reply:
I feel inheritance is not adding much value, and is creating issue for IDEs/tools.
Ultimately we should try to align all the TFMs by reducing the number of variations IMHO.